site/content/blog/2021/2021-11-22-Conatinerdevelopment.adoc

```diff
@@ -0,0 +1,80 @@
+= Remote Dev Environments
+:jbake-title: Remote Dev Environments
+:jbake-author: rdmueller
+:jbake-type: post
+:jbake-toc: true
+:jbake-status: published
+:jbake-tags: container, development
+:jbake-lang: en
+:doctype: article
+:toc: macro
+
+++++
+<style>
+.center {
+  text-align: center;
+}
+</style>
+++++
+
+Do you remember the times when the biggest challenge to compile a project was to set up the correct build environment with all required dependencies?
+
+The flexibility you have as a developer with current technologies is amazing.
+Not only that you can setup your dev environment with docker but you also chose where to run it.
+You can even choose to run your IDE on a remote machine.
+
+Let's take a look at the options you have with Visual Studio Code or short VS Code.
+
+Let's start with the standard way you use https://code.visualstudio.com/[VS Code] for development.
+You download and install it.
+Then you run it locally, clone your project from a git repository and develop locally.
+You have to ensure that everybody who uses the code has all needed external dependencies like build tools installed.
+
+[.center]
+image::blog/2021/vscode.dio.png[]
+
+That's ok, but let's take a look how we can help others to get the right start with the right environment for your code.
+
+== Move your environment to a container
+
+The next logical step is to move your environment to a container.
+
+The VS Code team started soon to experiment with splitting up the IDE into a frontend and a server backend.
+When you start VS Code on a Windows environment from within the Windows Subsystem for Linux (WSL), it just starts the backend server  in WSL and then connects to the frontend which runs natively on Windows.
```

Comment:
server  in WSL -> Leerzeichen zu viel.
it just starts -> Eventuell das _just_ entfernen?

Reply:
fixed. danke